src/utils/featheryClient/index.ts

```diff
@@ -552,13 +552,51 @@ export default class FeatheryClient extends IntegrationClient {
     const data = {
       fuser_key: userId,
       extraction_id: extractionId,
-      run_async: runAsync
+      run_async: true
```

Comment:
seems like we don't need this flag anymore?

Reply:
I was leaving it for backwards compatibility but we might not need to do that if no one is embedded using this feature right now